lib/commands/init.js

```diff
@@ -34,6 +37,27 @@ module.exports = Command.extend({
     }
   },
 
+  beforeRun: function(rawArgs) {
+    if (!rawArgs.length) {
+      return;
+    }
+    // merge in blueprint availableOptions
+    var blueprint;
+    try{
+      blueprint = this.lookupBlueprint(rawArgs[0]);
+      this.registerOptions(blueprint);
+    }
```

Comment:
These are all my stylistic preferences as well, but I basically copied this method from [here](https://github.com/ember-cli/ember-cli/blob/master/lib/commands/generate.js#L32). Should I prefer style over consistency here?